### ISSUE TITLE
notify IANAbis and ask review from IANA

### ISSUE THREAD

**Issue by thomas-fossati** [core-wg/cf-reg-update#4]
Note from @fpalombini:

* [ ] I think it is worth sending the doc to that wg [IANAbis] when it gets towards the latest stages.
* [x] It is also worth asking for an early review from IANA.

**Comment by thomas-fossati** [comment]
> Below is a (somewhat reworded & abbreviated) copy of the IANA review comments on -02:
> 
> 1. Proposal to add a "Notes" column as below:
> 
> Range | Registration Procedure | Notes 0-255 | Expert Review 256-9999 | IETF Review or IESG Approval 10000-64999 | Expert Review | Lightweight review described in RFC XXXX, Section 5.2.1 65000-65535 | Experimental Use | No operational use

This has been addressed in #16 

> 2. in Section 5.2.1, items 2 and 4 end with “… or IANA has approved its registration.” However, IANA isn’t the party that approves non-FCFS registrations. Media types are approved by either IESG-designated experts (vendor tree) or the IESG (standards tree, usually via the RFC publication process), while HTTP Content Codings require IETF Review (the IESG approves an IETF Stream I-D for publication as an RFC). In practical terms, if an expert approves a registration, IANA should add that entry to the registry within a day or two, and if the IESG approves a document for publication,
> 
> “… must exist in the registration in the [X] registry” could be changed to “… must have been approved for registration in the [X] registry”

This has been addressed in #12

**Comment by thomas-fossati** [comment]
I guess this can be closed since WGLC went (also) through IANAbis.

@EskoDijk wdyt?

**Comment by EskoDijk** [comment]
@thomas-fossati Yes ok for me; others in the CoRE interim also agreed to closing.
